presto-main/src/test/java/com/facebook/presto/sql/planner/TestEffectivePredicateExtractor.java

```diff
@@ -458,7 +458,102 @@ public void testInnerJoin()
                         equals(DE, EE),
                         lessThan(FE, bigintLiteral(100)),
                         equals(AE, DE),
-                        equals(BE, EE)));
+                        equals(BE, EE),
+                        lessThan(BE, EE)));
```

Comment:
```
          equals(BE, EE),
          lessThan(BE, EE)
```
This looks like `false`.

Reply:
That's true, but `EffectivePredicateExtractor` is not that clever right now. I will change to `lessOrEqual`...